### PR TITLE
feat: Added correct limits to all forwarded requests

### PR DESCRIPTION
**Pull request by mitsuhiko**
This adds limits to the forwarded requests. The test failures are consistent with https://github.com/getsentry/semaphore/pull/54

### CODE REVIEW

**Comment by mitsuhiko** [comment]
In the process I noticed we have a legacy endpoint `/api/store/`. I'm fine if we don't support it but that should be a conscious decision.

File: .travis.yml [hunk 1]
```diff
@@ -47,20 +45,24 @@ matrix:
...
-      env: SUITE=pytest
-      install: easy_install virtualenv
 
     # Integration tests
     - os: linux
-      env: SUITE=integration-test
-    - os: osx
-      env: SUITE=integration-test
-      install: easy_install virtualenv
+      env: PYTHON=python3.7 SUITE=integration-test
```
Comment (jan-auer) [inline .travis.yml]: _nit_: To simplify the setup, can we have the Makefile ensure a venv with the correct python version and instead use `language: rust` here? This should also make it easier to reproduce tests locally.

**Review by jan-auer — APPROVED** [review]

**Comment by jan-auer** [comment]
@mitsuhiko Apart from my comment above, good to merge 👍

**Review by untitaker — APPROVED** [review]
lgtm though I quite dislike that we vendor our own globbing logic

**Pull request merged**